src/components/KTreeList/KTreeItem.vue

```diff
@@ -3,14 +3,25 @@
     class="tree-item"
     :class="{
       'not-draggable': disabled,
-      'selected': item.selected
+      'selected': item.selected,
+      'expanded': collapsable && isExpanded,
+      'collapsed': collapsable && !isExpanded
     }"
     :data-testid="`tree-item-${item.id}`"
     data-tree-item-trigger="true"
     draggable="false"
     type="button"
     @click.prevent="handleClick"
   >
+    <component
+      :is="expandedIcon"
```

Comment:
<img width="260" alt="image" src="https://github.com/user-attachments/assets/711283d0-f64d-4a34-bfb2-e2712f4bcb3f" />

We need the icon to appear outside of the item, closer to as in the Figma